astropy/coordinates/representation.py

```diff
@@ -114,9 +119,22 @@ def get_name(cls):
             name = name[:-14]
         return name
 
-    def __getitem__(self, view):
-        return self.__class__(*[getattr(self, component)[view]
-                                for component in self.components])
+    def _replicate(self, method, *args, **kwargs):
```

Comment:
@StuartLittlefair - this is a leftover from `Time`, where I hijacked a `replicate` method which in fact did just replicate the instance (without copying the data). I'm quite happy to replace it with something else! Do you have a suggestion?